server/listeners/eventHandlers/domainEvents/index.ts

```diff
@@ -32,6 +32,9 @@ export default function buildEventHandler({ licenceService, prisonerService }: S
             offenderDetailsChangedEventHandler.handle(eventMessage).catch(error => logger.error(error))
           }
           break
+        default: {
+          // silently ignore
+        }
```

Comment:
one of the eslint rules is to warn if no default case in switch statements - this is generally a good thing. 

I've left the existing behaviour (ignore) for now but at least its explicit.